src/Microsoft.Azure.WebJobs.Host/ConsoleTraceWriter.cs

```diff
@@ -1,53 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Diagnostics;
-using System.IO;
-
-namespace Microsoft.Azure.WebJobs.Host
-{
-    /// <summary>
-    /// This <see cref="TraceWriter"/> delegates to a collection of inner user <see cref="TraceWriter"/>s (from <see cref="JobHostConfiguration.Tracing"/>,
-    /// as well as a Console <see cref="TextWriter"/>.
-    /// </summary>
-    internal class ConsoleTraceWriter : CompositeTraceWriter
```

Comment:
Where is console logging and coloration handled now?

Reply:
You now get the default ConsoleLogger that ships with .NET. It doesn't look quite as minimal as what we currently have, though. We should be able to tweak it and use something custom.